CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+Changes in 0.6.1 (2016-11-23)
+=============================
+
+New features:
+ - Storing IRC Passwords Encrypted at Rest: No longer will users need to `IDENTIFY` with NickServ! Simply send the admin room command `!storepass [server.name] password`, and your password that you would normally send to NickServ will be stored in the bridge database in an encrypted form (and not as a hash). WARNING: The bridge needs to send plaintext passwords to the IRC server when the bridge is connecting the user to IRC (typically on startup). This means that the key used to decrypt passwords must be loaded into memory on startup, and the key file will typically be stored on the same machine as the bridge itself. `!removepass [server.name]` will remove the password that the user has set from the database.
+
+Improvements:
+ - `!quit [server.name]` now attempts to kick the matrix user that issues the command from the rooms in which they are being briged. This is done after the user's respective IRC client is disconnected.
+ - The bridge now randomly jitters quit debounce delays between a minimum and maximum amount of time. This is in order to prevent the HS being sent many part requests all at once following a net-split that lasts a very long time. (See `quitDebounce` in config.sample.yaml)
+ - Errors received by the bridge when _joining_ an IRC client to a channel can now be seen in the admin room at startup.
+ - The bridge will now only claim for the namespaces pertaining to the HS connected, rather than any HS (which might also have an IRC bridge running).
```

Comment:
Wording feels wonky, you don't even mention whether this relates to users or rooms or both or neither. This is also a bug not an improvement.

> Previously the bridge may have treated users on other Homeservers as its own users. This meant they would not be bridged to IRC. The bridge will now respect the homeserver domain when determining whether it owns users or not.